tests/test_conn.py

```diff
@@ -130,33 +130,31 @@ def test_users_to_warn(conn, monkeypatch):
     weekuser = conn.add_email_account(weektoken)
     dayuser = conn.add_email_account(daytoken)
 
-    sysdate = int(time.time())
-    # add 20 hours to time
-    sysdate += parse_expiry_code("20h")
```

Comment:
This addition to the sysdate is now getting lost, but that is not a problem I guess?

Reply:
i didn't see subsequent tests depending on thi so it's ok i guess.